pkg/constants/http.go

```diff
@@ -106,6 +106,12 @@ func URLForGetConfig() string {
 	return GetConfig
 }
 
+// URLForGetClusterStatus returns url for GET request to find
+// status of the cluster
+func URLForGetClusterStatus() string {
+	return GetClusterStatus
+}
```

Comment:
This does nothing bu return a public constant... just use the constant

Reply:
Done